diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index 69644bc4..d68411c6 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -78,41 +78,3 @@ add_subdirectory(reflection)
 add_subdirectory(roundtrip)
 add_subdirectory(stencil)
 add_subdirectory(utility_formats)
-
-# We don't run find_package_test or glaze-install_test with MSVC/Windows, because the Github action runner often chokes
-# Don't run find_package on Clang, because Linux runs with Clang try to use GCC standard library and have errors before Clang 18
-if(CMAKE_CXX_COMPILER_ID MATCHES "GNU")
-    add_test(
-    NAME glaze-install_test
-    COMMAND
-    "${CMAKE_COMMAND}"
-    --install "${PROJECT_BINARY_DIR}"
-    --prefix "${CMAKE_CURRENT_BINARY_DIR}/install"
-    --config $<CONFIG>
-    --verbose
-    )
-
-    add_test(
-    NAME find_package_test
-    COMMAND
-    "${CMAKE_CTEST_COMMAND}"
-    --verbose
-    --output-on-failure
-    --build-noclean
-    --build-project "${PROJECT_NAME}" # helps msvc when --build-target
-    --build-generator "${CMAKE_GENERATOR}"
-    --build-config $<CONFIG>
-    --build-and-test
-    "${CMAKE_CURRENT_SOURCE_DIR}/find_package"
-    "${CMAKE_CURRENT_BINARY_DIR}/find_package"
-    --build-options
-    "-Dglaze_ROOT:PATH=${CMAKE_CURRENT_BINARY_DIR}/install"
-    "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
-    "-DBUILD_TESTING=ON"
-    --test-command "${CMAKE_CTEST_COMMAND}" --verbose --output-on-failure # inner ctest command
-    )
-
-    set_tests_properties(glaze-install_test PROPERTIES FIXTURES_SETUP glaze-install-fixture)
-    set_tests_properties(find_package_test PROPERTIES FIXTURES_REQUIRED glaze-install-fixture)
-endif()
-
